lib/Mojo/DOM.pm

```diff
@@ -468,6 +486,16 @@ excluded.
   # "foo\nbarbaz\n"
   $dom->parse("<div>foo\n<p>bar</p>baz\n</div>")->at('div')->all_text;
 
+=head2 each_text
+
+  my @text = $dom->each_text;
```

Comment:
Mojo::Headers's to_hash() can take "1" as an argument, so maybe we could add that to `all_text()` instead of adding a new method? Also, how about returning a Mojo::Collection?